Cue/app/common/SyncConflict.js

```diff
@@ -5,53 +5,111 @@ import { View, Text, ListView, Dimensions, Navigator, Platform, Alert} from 'rea
 import type { Deck } from '../api/types';
 import { connect } from 'react-redux'
 import CueIcons from './CueIcons'
+import CueColors from './CueColors'
 import CueHeader from './CueHeader'
 import { resolveConflict } from '../actions/library'
-import { SegmentedControls } from 'react-native-radio-buttons'
+import SelectableTextTableRow from './SelectableTextTableRow'
+import LibraryApi from '../api/Library'
+import TableHeader from './TableHeader'
 
-  //TODO: issue #146: style this page
+type Conflict = {
+  localDeck: ?Deck,
+  serverDeck: ?Deck,
+  change: {},
+  useServerDeck: ?boolean,
+}
 
 type Props = {
-  failedSyncs: Array<Deck>,
+  failedSyncs: Array<{}>,
   navigator: Navigator,
 
   // Redux
   localDecks: Array<Deck>,
-  resolveConflict: () => void
+  resolveConflict: (conflict: Conflict) => any
 }
 
 const styles = {
   container: {
     flex: 1,
-    backgroundColor: 'white',
+    backgroundColor: Platform.OS === 'android' ? 'white' : CueColors.coolLightGrey,
   },
+
+  headerText: {
+    paddingHorizontal: 16,
+    paddingTop: 16,
+    paddingBottom: 8,
+    color: CueColors.primaryText,
+    fontSize: Platform.OS === 'android' ? 16 : 17,
+  }
 }
 
 class SyncConflict extends React.Component {
   props: Props
 
   state: {
     dataSource: ListView.dataSource,
-    conflicts: [{localDeck: Deck, useServerDeck: Boolean}]
+    conflicts: Array<Conflict>
   }
 
   constructor(props: Props) {
     super(props)
     let ds = new ListView.DataSource({
       rowHasChanged: (r1, r2) => r1 != r2
     })
-    let conflicts = props.failedSyncs.map(change => {
-      return {
-        localDeck: props.localDecks.find(deck=> deck.uuid==change.uuid),
-        useServerDeck: undefined,
-      }
-    })
+    let conflicts = []
     this.state = {
       dataSource: ds.cloneWithRows(conflicts),
       conflicts,
     }
   }
 
+  _internetAlert = () => {
+    Alert.alert (
+      Platform.OS === "android" ? 'Failed to resolve conflicts' : 'Failed to Resolve Conflicts',
+      'Check your Internet connection and try again.')
+  }
+
+  componentDidMount() {
+    let ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 != r2
+    })
+    let conflicts = [];
+    let promises = []
+    this.props.failedSyncs.forEach(change => {
```

Comment:
Yeah, the screen should be super transient anyway, so a simple centred `ActivityIndicator` is probably enough.